src/filters/array-filters.js

```diff
@@ -6,11 +6,13 @@ var toArray = require('../directives/public/for')._postProcess
  * Limit filter for arrays
  *
  * @param {Number} n
+ * @param {Number} offset (Decimal expected)
  */
 
-exports.limitBy = function (arr, n) {
+exports.limitBy = function (arr, n, off) {
```

Comment:
one more thing: the comment above is `offset`. Let's be consistent and use `offset` in the code too. LGTM after that.